src/main/groovy/com/twcable/grabbit/jcr/ProtoPropertyDecorator.groovy

```diff
@@ -85,17 +85,26 @@ class ProtoPropertyDecorator {
 
 
     boolean isMultiple() {
-        return hasValues()
```

Comment:
It was changed due to the message restructure, but the issue @pgoodrich found could occur with the old line as well.